### PR TITLE
fix broken link

### PR DESCRIPTION
**Pull request by joshspicer**

### CODE REVIEW

File: _implementors/features-distribution.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ A local Feature is referenced in the devcontainer's `feature` object **relative
 
 Additional constraints exists when including local Features in a project:
 
-* The project must have a `.devcontainer/` folder at the root of the [**project workspace folder**](/docs/specs/devcontainer-reference.md#project-workspace-folder).
+* The project must have a `.devcontainer/` folder at the root of the [**project workspace folder**](/spec/#project-workspace-folder).
```
Comment (jkeech) [inline _implementors/features-distribution.md]: What is the final link you are trying to create? https://containers.dev/spec/#project-workspace-folder doesn't work for me.

**Review by jkeech — DISMISSED** [review]

File: _implementors/features-distribution.md [hunk 2]
```diff
@@ -149,7 +149,7 @@ A local Feature is referenced in the devcontainer's `feature` object **relative
 
 Additional constraints exists when including local Features in a project:
 
-* The project must have a `.devcontainer/` folder at the root of the [**project workspace folder**](/docs/specs/devcontainer-reference.md#project-workspace-folder).
+* The project must have a `.devcontainer/` folder at the root of the [**project workspace folder**](/implementors/spec/#project-workspace-folder).
```
Comment (samruddhikhandale) [inline _implementors/features-distribution.md:152]:
```suggestion
* The project must have a `.devcontainer/` folder at the root of the [**project workspace folder**](/_implementors/spec.md#project-workspace-folder).
```

This link doesn't work for me, probably we need to change to ☝️ ?

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]

File: _implementors/features-distribution.md (hunk 2 above)
Comment (joshspicer) [reply _implementors/features-distribution.md:152]: This is where i'm trying to link to :D 
https://containers.dev/implementors/spec/#project-workspace-folder

Comment (samruddhikhandale) [reply _implementors/features-distribution.md:152]: Huh 🤦‍♀️

**Review by samruddhikhandale — APPROVED** [review]

**Review by samruddhikhandale — APPROVED** [review]

**Pull request merged**